packages/webdriver/src/utils.js

```diff
@@ -302,7 +302,7 @@ export function environmentDetector ({ hostname, capabilities, requestedCapabili
  */
 export function getErrorFromResponseBody (body) {
     if (!body) {
-        return new Error('unknown error')
+        return new Error('Response has empty body')
```

Comment:
Sorry for double typing, from your tests I got that body can be either empty or does not exist. So, better change message to 'Response has either empty body or body doesn't exist'. What do you think?

Reply:
Instead of "or" we can throw different errors

Reply:
I think it would be great! Good proposal!